GwApiService/src/main/java/us/gameandwatching/gwapi/service/calc/EloCalculator.java

```diff
@@ -0,0 +1,81 @@
+package us.gameandwatching.gwapi.service.calc;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+import us.gameandwatching.gwapi.core.EloPair;
+import us.gameandwatching.gwapi.core.MatchResult;
+
+@Singleton
+public class EloCalculator {
+
+  @Inject
+  public EloCalculator() {
+
+  }
+
+  public EloPair updateEloRating(EloPair eloPair) {
+    double i1 = eloPair.getPlayer1Rating();
```

Comment:
I'm a bit more in the camp of the more descriptive variable names, even in mathy usages, though I don't have strong feelings about it